### ISSUE TITLE
Metazoan Anatomy Extension not accessible

### ISSUE THREAD

**Issue by matentzn** [obophenotype/upheno#252]
This clearly does not belong here, but since the issue arises when trying to run the upheno makefile: the x-metazoan-anatomy extension of GO has an outdated purl:
http://snapshot.geneontology.org/ontology/extensions/x-metazoan-anatomy.owl

I am not sure, but just removing the snapshot subdomain would make it resolve correctly.
http://geneontology.org/ontology/extensions/x-metazoan-anatomy.owl

**Comment by drseb** [comment]
@cmungall can you fix or should I move to https://github.com/OBOFoundry/purl.obolibrary.org ?

**Comment by cmungall** [comment]
Hi Nico!

Not totally following. Where abouts in the import chain do you see a hardcoding of the snapshot URLs? All the imports should use purls.

Btw, GO doesn't update x-metazoan-anatomy anymore, this is now bundled into go-plus

**Comment by matentzn** [comment]
Hey Chris! 

So, in the makefile, there is a target "mirror/go.owl" which refers to the purl: http://purl.obolibrary.org/obo/go/extensions/x-metazoan-anatomy
If you click on it, it will redirect you to snapshot.geneontology.org (see above). Whatever resource is located there, it is not possible to access it.

I am trying to recreate all the mappings of the phenotype ontologies, in particular mp-hp (view I think), but also just generally familiarising myself with upheno which is closely related to some other task I am working on. Who is actually responsible for doing that on your end and publishing the updates? It is just that some folks have been saying the the mapping are already quite outdated, and they are needed in some pipelines critical to them.

**Comment by cmungall** [comment]
You can blame anything that has gone stale on me.

We should replace the lexical mapping code to use ontobio and have a regular build of this. Can you make a separate ticket for this?

For the inferred part, we still have this outstanding issue
https://github.com/obophenotype/human-phenotype-ontology/issues/1005

**Comment by matentzn** [comment]
Yeah no problem. There are some more issues with the pipeline (inconsistent ontologies and such like), so I will open tickets once I know what needs doing. Sorry for being slow on this, I am looking at a bit too many things this week :P I will check your patch at some point this week.
